common/constants/default.go

```diff
@@ -64,7 +64,7 @@ const (
 	DefaultVolumeStatsAggPeriod        = time.Minute
 	DefaultTunnelPort                  = 10004
 
-	CurrentSupportK8sVersion = "v1.19.3"
```

Comment:
This means the k8s version in KubeEdge which is v1.21.4 now, we can cherry-pick to release-1.9.

Reply:
got it, it's from vendor version.